src/graphql/server.js

```diff
@@ -58,5 +58,10 @@ server.applyMiddleware({ app });
 const handler = serverless(app);
 
 export const graphql = async (event, context) => {
+  // Prevents Lambda cold starts
+  if (event.source === 'serverless-plugin-warmup') {
+    return callback(null, 'Lambda is warm!');
```

Comment:
There’s no `callback` defined, so I think we can just `console.log` instead.

Reply:
@samovertonjr yup! That's the API for Lambdas, callback is the 3rd argument.

Reply:
or
`export const graphql = async (event, context, callback) => {
     if (event.source === 'serverless-plugin-warmup') {
       console.log('Lambda is warm');`

Reply:
You can also do something like this, I believe.

```js
export const graphql = async (event, context) => {
  if (event.source === 'serverless-plugin-warmup') {
    return 'Lambda is warm'
  }
}
```

per [this article](https://aws.amazon.com/blogs/compute/node-js-8-10-runtime-now-available-in-aws-lambda/)